internal/util/rocksmq/server/rocksmq/rocksmq_impl.go

```diff
@@ -276,6 +276,11 @@ func (rmq *rocksmq) CreateTopic(topicName string) error {
 	}
 	start := time.Now()
 
+	// Check if topicName contains "/"
+	if strings.Contains(topicName, "/") {
+		return fmt.Errorf("topic name = %s contains \"/\"", topicName)
+	}
```

Comment:
Maybe it's better to add some error log here. Just check the invocation chain. All the error handling does not print error log until the retry failed.
Also, the returned err shall be able to check type or a `retry.Unrecoverable` error to stop the retry backoff procedure.